site/src/components/link-area-delegation-explainer.md

```diff
@@ -0,0 +1,178 @@
+
+Despite this long-standing rule, there are websites which ignore it. In doing so, these sites end up with UI which will be difficult, if not blocking, for people using assistive technology, such as screen readers. On the flip-side, there are also websites such as [Walmart](https://walmart.ca), [Reddit](https://reddit.com), [Rotten Tomatoes](https://rottentomatoes.com), to name a few prominent ones, that use specific markup patterns and CSS or JS to present content that visually appears to have nested targets, as the larger container appears to be clickable - but inspecting the markup patterns would show otherwise.
+
+## The clikcable container or "card" pattern
+Commonly a clickable container, often referred to as a "card", will present a graphic or other media, text, a primary call to action (link or button) as well as other supplementary interactive elements:
+
+<figure>
+  <img src="https://github.com/user-attachments/assets/27e5076d-bc98-4fb8-a429-e8462ff9024c" 
+       width="400" 
+       alt="Screenshot of a Reddit topic 'card' from the r/cats subreddit. Shows time of publication, r/cats as a hyperlink, a join and more options button, followed by the title of the post and a graphic of a cat pearing out a window while its owner takes its picture. up/down vote, comment, award and share buttons follow the graphic.">
+
+  <figcaption>
+    <p>With this Reddit example, the whole card is clickable and doing so and would navigate a user to the Reddit post. But there are also other interactive elements (up/down vote, comments, join, the sub-reddit link, etc.) are also clickable. These appear to be "nested" within the clickable card, but due to work arounds, this is not the case.</p>
+  </figcaption>
+</figure>
+
+### Existing ways to achieve clickable containers
+Today, making a container/card clickable can be achieved by a few different approaches, each with its own tradeoffs:
+1. Using the `a` element to wrap the content that needs to be clickable.  The issue with this is:
+  a. doesn't support nested interactive elements without creating accessibility issues.
+  b. using an `a` element to wrap content is not supported in certain contexts - e.g., such as trying to wrap a row within a table.  The HTML parser will remove such hyperlinks from the rendered table markup.
+  c. even without nested interactive elements, the accessible name of the hyperlink can become overly verbose, or may not be exposed as all depending on the element the hyperlink wraps.
```

Comment:
i decided not to expand on this point beyond what i wrote here.  but maybe others might be curious / it might be a necessary rebuttal to any future feedback of "HTML already partially allows for this by wrapping content in a hyperlink".